FaunaDB.Client/FaunaDB.Client.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net45;netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
-    <Version>3.0.0</Version>
+    <Version>4.0.0-rc</Version>
     <PackageId>FaunaDB.Client</PackageId>
```

Comment:
dumb question here: is the `rc` the same purpose of "snapshots" for Maven repo (JVM) ? Can we override `rc` releases?

Reply:
I think they have different meanings. Maven's snapshot version is not considered a release version and can be updated, so a user who uses 3.0.0-snapshot, for example, can expect that there can be new changes within this same version.
Nuget's `rc` can't be changed in the same way, if we have new changes we need to publish another package, with, say, version 4.0.0-rc1, 4.0.0-rc2, and so on.

We can write anything after the "-", but conventionally we use alpha, beta, rc if we mean pre-release version. There should be a checkbox in visual studio to include pre-release version, so users can use them in their projects.
And they are also marked as pre-release on nuget.org:
https://www.nuget.org/packages/FaunaDB.Client/1.2.1-alpha